src/clientdlg.cpp

```diff
@@ -44,12 +44,11 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
     ClientSettingsDlg   ( pNCliP, pNSetP, parent ),
     ChatDlg             ( parent ),
     ConnectDlg          ( pNSetP, bNewShowComplRegConnList, parent ),
-    AnalyzerConsole     ( pNCliP, parent ),
-    MusicianProfileDlg  ( pNCliP, parent )
+    AnalyzerConsole     ( pNCliP, parent )
+//    MusicianProfileDlg  ( pNCliP, parent )
```

Comment:
Please go ahead and delete code.  Git makes it straight forward to revert changes.

Reply:
Oh, sorry, I missed that one.  I tried to clean up all such things which I, of course, use liberally at times.
I hope their aren't too many more.